config/samples/resources/bigqueryconnectionconnection/cloudresource-connection/bigqueryconnectionconnection_v1beta1_bigqueryconnectionconnection.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: bigqueryconnection.cnrm.cloud.google.com/v1beta1
+kind: BigQueryConnectionConnection
+metadata:
+  name: bigqueryconnection-sample-cloudresource
```

Comment:
Again, just want to double check, is `bigqueryconnection-sample-cloudresource` a valid resource name in BigQuery Connection API?